source/intro.Rmd

```diff
@@ -70,149 +63,206 @@ the methods described in this book:
-medicine CCC: If you do decide to continue research on CCC, (a) you may,
-or (b) you may not, come up with an important general cure within, say,
-the next 3 years. If you do come up with such a general cure, of course
-it will have very great social benefits. Furthermore, (c) if you decide
-not to do further research on CCC now, you can direct your time and that
-of other people to research in other directions, with some chance that
-the other research will produce a less-general but nevertheless useful
-cure for some relatively infrequent forms of cancer. Those three
-possibilities have different social benefits. The probability that
-medicine CCC really has some curative effect on cancer, as judged by
-your prior research, obviously will influence your decision on whether
-or not to do more research on medicine CCC. But that judgment about the
-probability is only one part of the overall web of consequences and
-evaluations that must be taken into account when making your decision
-whether or not to do further research on medicine CCC.
-
-Why does this book limit itself to the specific probability questions
-when ultimately we are interested in decisions? A first reason is
-division of labor. The more general aspects of the decision-making
-process in the face of uncertainty are treated well in other books. This
-book's special contribution is its new approach to the crucial process
-of estimating the chances that an event will occur.
+Continuing with the decision concerning whether to do more research on medicine
+AntiAnyVir: If you do decide to continue research on AntiAnyVir, (a) you may,
+or (b) you may not, come up with an important general treatment for viral
+infections within, say, the next 3 years. If you do come up with such a general
+treatment, of course it will have very great social benefits. Furthermore, (c)
+if you decide not to do further research on AntiAnyVir now, you can direct your
+time and that of other people to research in other directions, with some chance
+that the other research will produce a less-general but nevertheless useful
+treatment for some relatively infrequent viral infections. Those three
+possibilities have different social benefits. The probability that medicine
+AntiAnyVir really has some benefit in treating COVID19, as judged by your prior
+research, obviously will influence your decision on whether or not to do more
+research on medicine AntiAnyVir. But that judgment about the probability is
+only one part of the overall web of consequences and evaluations that must be
+taken into account when making your decision whether or not to do further
+research on medicine AntiAnyVir.
+
+Why does this book limit itself to the specific probability questions when
+ultimately we are interested in decisions? A first reason is division of
+labor. The more general aspects of the decision-making process in the face of
+uncertainty are treated well in other books. This book's special contribution
```

Comment:
Should we give them some references?

Reply:
We should - but I don't know what books to suggest.